src/widgets/refinement-list/refinement-list.tsx

```diff
@@ -24,7 +24,7 @@ const withUsage = createDocumentationMessageGenerator({
 const suit = component('RefinementList');
 const searchBoxSuit = component('SearchBox');
 
-type RefinementListOwnTemplates = {
+export type RefinementListOwnTemplates = {
```

Comment:
what is this exported for? shouldn't most use cases use `RefinementListTemplates`, which is slightly lower

Reply:
It is used to type `templateProps` here https://github.com/algolia/instantsearch.js/pull/4702/files#diff-000e299be1cea0f8ffc1aae9af4be59c88914c2d4c8fb5c0d80975ff85c2448fR55

Reply:
(it is no longer exported since `RefinementList` is now set as a Generic)